README.md

```diff
@@ -1,2 +1,55 @@
-# openbsd-packer-template
-Packer template for OpenBSD images for Vagrant and GCP
+# OpenBSD Packer template
+
+## Goal
+
+Goal of this project is to provide a small OpenBSD Vagrant and GCP images, primarily for testing
+Ansible roles.
+
+Vagrant Box is available here: https://app.vagrantup.com/devinsolutions/boxes/openbsd-66-basic
+
+By "basic" we mean that only `bsd`, `bsd.rd`, `base` and `comp` sets are included (plus GCE daemons
+and Python needed by them in GCE images). We support only latest stable OpenBSD release. No binary
+patches are included.
```

Comment:
I don't want to have half-patched images, therefore I see only two options: either include no binary patches, or update these images as soon as those patches become available. Since I have no automation for this set up right now and I don't want to update these images manually every time, I've decided not to include these patches.

Primary purpose is testing Ansible roles anyway, and one can install these patches before the test begins (e.g. in Molecule prepare playbook) if needed.

Reply:
I think that unpatched images are worse than half-patched. How about we provide up-to-date images on best-effort basis and share the responsibility for updating them among the whole admin team? Combined with installing patches before running the tests, we should be always secure while achieving the best performance to cost ratio.

Reply:
That sounds good. I've added a provisioner which installs all available binary patches.